### DISCUSSION TITLE
When I remove edges or vertices from janusgraph, the size of the table in Hbase does not change

### DISCUSSION THREAD

**Discussion by w924490509** [JanusGraph/janusgraph#3271] [Q&A]
I use Hbase as the storage backend for janusgraph, and when I delete the edges I see that the size of the Hbase table has not changed even after several hours.
`g.E().drop().iterate();`
This is my configuration:
`
gremlin.graph=org.janusgraph.core.JanusGraphFactory
storage.backend=hbase
storage.hostname=node1,node2,node3
storage.hbase.ext.zookeeper.znode.parent=/hbase
storage.hbase.table=GRAPH_ALL_0517
storage.hbase.region-count=60
cache.db-cache = true
cache.db-cache-clean-wait = 5
cache.db-cache-time = 10000
cache.db-cache-size = 0.3
cache.tx-cache-size=1000000
cache.tx-dirty-size=1000000
storage.batch-loading=true
schema.default=none
ids.block-size=50000000
ids.renew-timeout=3600000
ids.wait-time=600
ids.renew-percentage=0.5
id.num-partitions=20
storage.buffer-size=10000
graph.set-vertex-id=true
cluster.max-partitions=65535
force-index=true
`

**Comment by vtslab** [comment]
Things to check:
- did you commit the transaction with the drop() step?
- did you inspect the right table (something with edgestore, JanusGraph has multiple tables)

**Reply by w924490509** [reply]
I have confirmed what you said and there is no problem.
And I found that the number of rows in Hbase had changed, but the disk space was not released.

**Reply by vtslab** [reply]
OK, that makes sense. HBase has an automatic compaction process, so I guess releasing disk space is postponed until the compaction.

**Reply by w924490509** [reply]
It is a compression problem of Hbase. Thanks
